src/main/java/com/socrata/datasync/publishers/GISPublisher.java

```diff
@@ -205,15 +206,16 @@ private static String[] getStatus(URI uri,
 
     private static Blueprint postRawFile(URI uri,
                                          File file,
-                                         UserPreferences userPrefs) throws IOException {
+                                         UserPreferences userPrefs,
+                                         int timeoutMS) throws IOException {
         HttpUtility httpUtility = new HttpUtility(userPrefs, true, 3, 2);
 
         System.out.println("Posting file...");
         HttpEntity httpEntity = MultipartEntityBuilder.create()
             .addBinaryBody(file.getName(), file, ContentType.APPLICATION_OCTET_STREAM,file.getName())
             .build();
 
-        HttpResponse response = httpUtility.post(uri, httpEntity);
+        HttpResponse response = httpUtility.post(uri, httpEntity, timeoutMS);
```

Comment:
What was the default timeout before this change?

Reply:
1 minute (it was using the default timeouts set above)